app/models/edition.rb

```diff
@@ -88,6 +89,7 @@ def change_note_html
   def draft_copy
     dup.tap do |e|
       e.change_note = nil
+      e.change_reason_for_change = nil
```

Comment:
I'm a liar. `Edition#draft_copy` is used in `Guide#latest_editable_edition`. I'll delete the other method separately.